Where to access RUM and update links (#17896)
DataDog/documentation: content/en/real_user_monitoring/_index.md

## Explore Datadog RUM

> RUM can be accessed from [**UX Monitoring > RUM Applications**][1].

Reviewer: ```suggestion
Access RUM by navigating to [**UX Monitoring > RUM Applications**][1].
```
